python/lsst/ts/atmcssimulator/mcs_csc.py

```diff
@@ -47,23 +48,27 @@ class ATMCSCsc(attcpip.AtTcpipCsc):
 
     def __init__(
         self,
-        config_dir: str | None = None,
+        config_dir: str | pathlib.Path | None = None,
+        check_if_duplicate: bool = False,
         initial_state: salobj.State = salobj.State.STANDBY,
+        override: str = "",
+        simulation_mode: int = 0,
     ) -> None:
         super().__init__(
             name="ATMCS",
             index=0,
             config_schema=CONFIG_SCHEMA,
             config_dir=config_dir,
             initial_state=initial_state,
-            simulation_mode=1,
+            override=override,
+            simulation_mode=simulation_mode,
         )
 
         # McsSimulator for simulation_mode == 1.
         self.simulator: McsSimulator | None = None
 
     async def start_clients(self) -> None:
-        if self.simulator is None:
+        if self.simulation_mode == 1 and self.simulator is None:
```

Comment:
Maybe:

```
if (self.simulation_mode == 1) and (self.simulator is None):
```

Reply:
I know that increases readability but the parenthesis are not necessary. I prefer the version without them.